app/models/organization.js

```diff
@@ -0,0 +1,18 @@
+import Model, { attr, hasMany, belongsTo } from '@ember-data/model';
+
+export default class OrganizationModel extends Model {
+  @attr name;
```

Comment:
```suggestion
  @attr naam;
```
See comment about consistency here https://github.com/lblod/app-subsidiepunt/pull/22/files#r1683192111